### ISSUE TITLE
Still issues with coveralls reporting

### ISSUE THREAD

**Issue by jessesquires** [Instagram/IGListKit#227]
Ok, so `slather` is fine now. But it looks like coveralls is still having problems.

https://travis-ci.org/Instagram/IGListKit/jobs/177146187#L370-L394

```bash
cd $TRAVIS_BUILD_DIR && slather
Slathering...
No coverage directory found.
	Are you sure your project is generating coverage? Make sure you enable code coverage in the 

Test section of your Xcode scheme.
	Did you specify your Xcode scheme? (--scheme or 'scheme' in .slather.yml)
	If you're using a workspace, did you specify it? (--workspace or 'workspace' in .slather.yml)

/Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/slather-2.3.0/lib/slather/project.rb:177:in `profdata_coverage_dir': No coverage directory found. (StandardError)
```

**Comment by jessesquires** [comment]
The `IGListKit` scheme "generate coverage" box is checked.

Am I missing something obvious @rnystrom ?

Also -- our `slather.yml` specifies *both* a project and a workspace:
https://github.com/Instagram/IGListKit/blob/master/.slather.yml#L3-L4

Is that correct?

**Comment by rnystrom** [comment]
Recapping events:

So the last build that succeeded w/ coveralls support was:

https://travis-ci.org/Instagram/IGListKit/jobs/176428331

The next commit was 879260b417fe67ac4b4edc9b04f7729949f3634f which broke builds. Fixed in d6cae084bb1565dcdf9afb7e96d4f0e80ba6e4f0

We got rid of the Gemfile and bundler. Slather was missing, added back in 8c32f2581fcfb19d070d94b76b298799bbf913c8.

Travis runs `gem install slather --no-rdoc --no-ri --no-document --quiet` which build output shows still installs `2.3.0`.

[This build job](https://travis-ci.org/Instagram/IGListKit/jobs/177139465) has build failures w/ the TV tests:

```
❌  Undefined symbols for architecture x86_64
> Symbol: _OBJC_CLASS_$_IGListTestAdapterStoryboardDataSource
> Referenced from: objc-class-ref in IGListAdapterStoryboardTests.o
❌  ld: symbol(s) not found for architecture x86_64
❌  clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

I know I had to add `cd $TRAVIS_BUILD_DIR` before `slather` even though most projects don't seem to use it, I figured that was b/c of bundler, but I'm not sure. We could try removing that.

I'll put up a PR that fixes the TV tests first and see if that helps.
